packages/financial-templates-lib/test-truffle/BalancerPriceFeed.js

```diff
@@ -1,9 +1,9 @@
 const { toWei, toBN } = web3.utils;
 const winston = require("winston");
 
-const { BalancerPriceFeed } = require("../../src/price-feed/BalancerPriceFeed");
```

Comment:
pretty sure you did not mean to change these. likely want to revert them I'm guessing?

Reply:
it's correct! it's because to create the parallelism on tests, was necessary split the test folder in `test` and `tests-truffle` to run with truffle and hardhat in different jobs, so the required files change the folder too.